ignite/distributed/utils.py

```diff
@@ -350,6 +353,68 @@ def all_reduce(
     return _model.all_reduce(tensor, op, group=group)
 
 
+def all_gather_tensors_with_shapes(
```

Comment:
You do not expose this method as `idist.all_gather_tensors_with_shapes` ?

Reply:
It is exposed. Isn't? It's in `__all__` attr of module.